recipes/braker2/path.patch

```diff
@@ -1,62 +1,111 @@
+-#!/usr/bin/env perl
 +#!/opt/anaconda1anaconda2anaconda3/bin/perl
  
  # Author: Katharina J. Hoff and Mario Stanke
  # Contact: katharina.hoff@uni-greifswald.de
---- scripts/filterGenemark.pl	2018-09-10 08:36:32.000000000 -0500
-+++ scripts/filterGenemark.pl.new	2019-04-13 00:56:53.596590032 -0500
+diff --git a/scripts/filterGenemark.pl b/scripts/filterGenemark.pl
+index 9f75b44..5b8e464 100755
+--- a/scripts/filterGenemark.pl
++++ b/scripts/filterGenemark.pl
+@@ -1,4 +1,4 @@
+-#!/usr/bin/env perl
++#!/opt/anaconda1anaconda2anaconda3/bin/perl
+ 
+ ####################################################################################################
+ #                                                                                                  #
+diff --git a/scripts/filterIntronsFindStrand.pl b/scripts/filterIntronsFindStrand.pl
+index d489dec..be55264 100755
+--- a/scripts/filterIntronsFindStrand.pl
++++ b/scripts/filterIntronsFindStrand.pl
 @@ -1,4 +1,4 @@
 -#!/usr/bin/env perl
 +#!/opt/anaconda1anaconda2anaconda3/bin/perl
  
  ####################################################################################################
  #                                                                                                  #
---- scripts/filterIntronsFindStrand.pl	2018-09-10 08:36:32.000000000 -0500
-+++ scripts/filterIntronsFindStrand.pl.new	2019-04-13 00:59:33.829557574 -0500
+diff --git a/scripts/filter_augustus_gff.pl b/scripts/filter_augustus_gff.pl
+index c06e766..414320d 100755
+--- a/scripts/filter_augustus_gff.pl
++++ b/scripts/filter_augustus_gff.pl
 @@ -1,4 +1,4 @@
 -#!/usr/bin/env perl
 +#!/opt/anaconda1anaconda2anaconda3/bin/perl
  
  ####################################################################################################
  #                                                                                                  #
---- scripts/findGenesInIntrons.pl	2018-09-10 08:36:32.000000000 -0500
-+++ scripts/findGenesInIntrons.pl.new	2019-04-13 00:58:55.047565429 -0500
+@@ -141,4 +141,4 @@ if(defined($out_file)){
+ }else{
+ 	my $n_tx = keys %keep;
+ 	print "$n_tx\n";
+-}
+\ No newline at end of file
++}
+diff --git a/scripts/findGenesInIntrons.pl b/scripts/findGenesInIntrons.pl
+index 0cf4476..450f0a4 100755
+--- a/scripts/findGenesInIntrons.pl
++++ b/scripts/findGenesInIntrons.pl
 @@ -1,4 +1,4 @@
--#!/usr/bin/perl
+-#!/usr/bin/env perl
 +#!/opt/anaconda1anaconda2anaconda3/bin/perl
  # ==============================================================
  # Katharina J. Hoff
  # When two gene sets are merged with joingenes, genes that
---- scripts/startAlign.pl	2018-09-10 08:36:32.000000000 -0500
-+++ scripts/startAlign.pl.new	2019-04-13 00:58:15.723573394 -0500
+diff --git a/scripts/gatech_pmp2hints.pl b/scripts/gatech_pmp2hints.pl
+index 22fc880..a00edd5 100755
+--- a/scripts/gatech_pmp2hints.pl
++++ b/scripts/gatech_pmp2hints.pl
+@@ -1,4 +1,4 @@
+-#!/usr/bin/env perl
++#!/opt/anaconda1anaconda2anaconda3/bin/perl
```

Comment:
Why is this changed?

Reply:
So it uses the perl in the same environment.

Reply:
A normal shebang also like the above will also use the same perl in the same environment if activated properly. Can you point me to this conda-forge recommendation? I don't think we should change code to make it anaconda compatible :( - However, we do encourage people since years to use a proper shebang. E.g. `/usr/bin/env perl` vs. `/usr/bin/perl`.

Reply:
If environments are stacked then you otherwise get the perl from the last one activated, which may not be correct. When I come across people recommending this modified shebang again i'll let you know, I don't know that it was ever codified.